### PR TITLE
update documentation for visualize_colorspace

### LINKED ISSUE #893: Outdated doc page for pcv.visualize.colorspaces
**Describe the bug**
I noticed that the doc page for the pcv.visualize.colorspaces is missing documentation for the `original_img` parameter. This is param is optional since it has a default value. 

**Additional context**
Also needs updating in the `updating.md` page too.

### PR DESCRIPTION
**Pull request by jdavidpeery**
**Describe your changes**
added original_img optional parameter to documentation in visualize_colorspace.md and updating.md

**Type of update**
Is this a:
* Update to documentation

**Associated issues**
closes #893 

**Additional context**

### CODE REVIEW

**Review by HaleySchuhl — APPROVED** [review]
Thanks @jdavidpeery for the speedy update!

**Comment by nfahlgren** [comment]
Arguments with default values should have the default value included in the function docs, e.g.:

```
pcv.visualize.colorspaces(rgb_img, original_img=True)
```

**Comment by nfahlgren** [comment]
Not in the example code but at the top of the doc page and in the updates doc

File: docs/updating.md [hunk 1]
```diff
@@ -951,7 +951,7 @@ pages for more details on the input and output variable types.
 #### plantcv.visualize.colorspaces
 
 * pre v3.8: NA
-* post v3.8: plotting_img = pcv.visualize.colorspaces(rgb_img)
+* post v3.8: plotting_img = pcv.visualize.colorspaces(rgb_img, original_img)
```
Comment (jdavidpeery) [inline docs/updating.md:954]: @nfahlgren I thought I had done that here

File: docs/visualize_colorspace.md [hunk 2]
```diff
@@ -2,12 +2,13 @@
 
 This is a plotting method used to examine all potential colorspaces from available PlantCV functions.
 
-**plantcv.visualize.colorspaces**(*rgb_img*)
+**plantcv.visualize.colorspaces**(*rgb_img, originial_img*)
```
Comment (jdavidpeery) [inline docs/visualize_colorspace.md:5]: and here, is there somewhere else you meant?

File: docs/updating.md (hunk 1 above)
Comment (nfahlgren) [reply docs/updating.md:954]: That's just the keyword though, since it has a default value it should be `original_img=True`

Comment (nfahlgren) [reply docs/updating.md]: Yep!

File: docs/visualize_colorspace.md (hunk 2 above)
Comment (HaleySchuhl) [reply docs/visualize_colorspace.md]: Ya! Just have to include the default param value too. `originial_img = True*`

**Pull request merged**